npm/ipsm/ipsm.go

```diff
@@ -188,20 +188,24 @@ func (ipsMgr *IpsetManager) DeleteFromList(listName string, setName string) erro
 	exists, _ := ipsMgr.SetExists(setName)
 
 	// if set does not exist, then return because the ipset call will fail due to set not existing
+	// TODO make sure these are info and not errors, use NPmErr
 	if !exists {
-		return fmt.Errorf("Set [%s] does not exist when attempting to delete from list [%s]", setName, listName)
+		metrics.SendErrorLogAndMetric(util.IpsmID, "Set [%s] does not exist when attempting to delete from list [%s]", setName, listName)
```

Comment:
(Log collection question) Do we need to collect these information? If it is ignorable case (especially, it returns "nil"), are there special reason to send this log?

Reply:
For debugging these messages can be very useful. It gives a full picture of why the deleteFromList failed or was a no-op.  Because NPM cases are sometimes hard to debug, any additional log message is useful